native.ts

```diff
@@ -40,3 +40,6 @@ export const RNZoomUsVideoView = (
 ) as HostComponent<NativeVideoProps> | null
 
 export const RNZoomUs = NativeModules.RNZoomUs
+if (!RNZoomUs) console.error('[react-native-zoom-us] RNZoomUs native module is not linked.')
+
+export default RNZoomUs
```

Comment:
Yeap, would be great to remove this.

Reply:
Ok. I removed.

But it is weird for me. Usually we do vice versa, and add maximum possible imports. 
And default import as a requirement :)

Reply:
Thnx.
:thinking: from my experience, named exports are better for auto-complete and easier to use since you do not need to think about how you will name the default export.